src/transports/sentry_libcurl_transport.c

```diff
@@ -95,8 +98,10 @@ header_callback(char *buffer, size_t size, size_t nitems, void *userdata)
     if (sep) {
         *sep = '\0';
```

Comment:
hm, I think when using slices, you can avoid this mutation, and the copy of `buffer`, but meh…

Reply:
Yeah. This can be improved.